src/commands/initProjectForVSCode/verifyVSCodeConfig.ts

```diff
@@ -138,37 +140,136 @@ async function verifyJavaDeployConfigIsValid(projectLanguage: ProjectLanguage |
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+interface IVerifyFrameworkResult {
+    mismatchTargetFramework: string;
+    update(): Promise<void>;
+}
+
+// https://docs.microsoft.com/dotnet/standard/frameworks
+const targetFrameworkRegExp: RegExp = /net(standard|coreapp)?[0-9.]+/i;
+
+function verifyTasksFramework(workspacePath: string, projTargetFramework: string): IVerifyFrameworkResult | undefined {
+    const config: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration('tasks', vscode.Uri.file(workspacePath));
+    const tasks: ITask[] | undefined = config.get<ITask[]>('tasks');
+    if (tasks) {
+        let mismatchTargetFramework: string | undefined;
+
+        for (const task of tasks) {
+            if (task.options && task.options.cwd) {
+                const matches: RegExpMatchArray | null = task.options.cwd.match(targetFrameworkRegExp);
+                const targetFramework: string | null = matches && matches[0];
+                if (targetFramework && targetFramework.toLowerCase() !== projTargetFramework.toLowerCase()) {
+                    mismatchTargetFramework = targetFramework;
+                    task.options.cwd = task.options.cwd.replace(targetFramework, projTargetFramework);
+                }
+            }
+        }
+
+        if (mismatchTargetFramework) {
+            return {
+                mismatchTargetFramework,
+                update: async (): Promise<void> => {
+                    config.update('tasks', tasks);
+                }
+            };
+        }
+    }
+
+    return undefined;
+}
+
+function verifySettingsFramework(workspacePath: string, projTargetFramework: string): IVerifyFrameworkResult | undefined {
+    let deploySubPath: string | undefined = getFuncExtensionSetting(deploySubpathSetting, workspacePath);
+    if (deploySubPath) {
+        const matches: RegExpMatchArray | null = deploySubPath.match(targetFrameworkRegExp);
+        const targetFramework: string | null = matches && matches[0];
+        if (targetFramework && targetFramework.toLowerCase() !== projTargetFramework.toLowerCase()) {
+            deploySubPath = deploySubPath.replace(targetFramework, projTargetFramework);
+            return {
+                mismatchTargetFramework: targetFramework,
+                update: async (): Promise<void> => {
+                    await updateWorkspaceSetting(deploySubpathSetting, deploySubPath, workspacePath);
+                }
+            };
+        }
+    }
+
+    return undefined;
+}
+
 async function verifyPythonVenv(projectPath: string, actionContext: IActionContext): Promise<void> {
     const venvName: string | undefined = getFuncExtensionSetting(pythonVenvSetting, projectPath);
     if (venvName && !await fse.pathExists(path.join(projectPath, venvName))) {
-        actionContext.properties.pythonVenvExists = 'false';
+        actionContext.properties.verifyConfigPrompt = 'createVenv';
```

Comment:
Are these all exclusive telemetry events?  It seems like they're getting overwritten (such as `createVenv` and `update`) but I could be totally off here.